target/linux/ath79/dts/ar7161_dlink_dir-825-b1.dts

```diff
@@ -198,10 +214,9 @@
 				read-only;
 			};
 
-			partition@670000 {
-				label = "unknown";
+			fwconcat1: partition@670000 {
+				label = "fwconcat1";
 				reg = <0x670000 0x190000>;
-				read-only;
```

Comment:
The final 64kb is a backup of `caldata` partition, It's better to keep it. Please refer to https://github.com/openwrt/openwrt/pull/9401

Reply:
this is cal data is only a product of using the ar71xx BIG
none of  dir-825-b1 devices had a copy of cal data there

Reply:
Thanks for your explanation, I get it. https://github.com/openwrt/openwrt/commit/ffc0ea6c5d1ffe41a794926ba3b4a794f9833406